tests/unit_tests/segmentation_processing_PadShortToCropSize_test.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+import numpy as np
+
+from super_gradients.training.processing import SegmentationPadShortToCropSize
+from super_gradients.training.utils.predict.predictions import SegmentationPrediction
+
+
+class SegmentationPadShortToCropSizeTest(unittest.TestCase):
```

Comment:
Please also add a test case for segmentation predict here https://github.com/Deci-AI/super-gradients/blob/master/tests/unit_tests/test_predict.py

Reply:
done (4df033eecccf78c3cd10dfe89771242bd3c0abcf)